src/proxy/topic/ReliableTopicProxy.ts

```diff
@@ -204,4 +268,22 @@ export class ReliableTopicProxy<E> extends BaseProxy implements ITopic<E> {
         }).catch(deferred.reject);
     }
 
+    private addMessagesOrFail(messages: ReliableTopicMessage[]): Promise<void> {
+        return this.ringbuffer.addAll(messages, OverflowPolicy.FAIL).then((sequenceId: Long) => {
+            if (sequenceId.toNumber() === -1) {
+                throw new TopicOverloadError(`Failed to publish messages on topic ${this.name}.`);
+            }
+        });
+    }
+
+    private addAndBlock(messages: ReliableTopicMessage[], pauseMillis: number): Promise<void> {
+        this.ringbuffer.addAll(messages, OverflowPolicy.FAIL).then((id: Long) => {
+            if (id.toNumber() === -1) {
+                setTimeout(() => {
+                    this.addAndBlock(messages, Math.min(pauseMillis * 2, 2000));
```

Comment:
```suggestion
                    this.addAndBlock(messages, Math.min(pauseMillis * 2, MAX_BACKOFF));
```

Reply:
Done.

Reply:
🤔 are you sure? I will do this myself as well.